bmo_install_prereqs.sh

```diff
@@ -0,0 +1,13 @@
+# Install operator-sdk for use by the baremetal-operator
```

Comment:
This could probably go in 03_ocp_repo_sync.sh where we clone various other things, fine to leave it as a separate file if you prefer though